alot/db/thread.py

```diff
@@ -152,8 +152,8 @@ def get_authors(self):
         if self._authors is None:
             seen = {}
             msgs = self.get_messages().keys()
-            msgs_with_date = filter(lambda m: m.get_date() is not None, msgs)
-            msgs_without_date = filter(lambda m: m.get_date() is None, msgs)
+            msgs_with_date = list(filter(lambda m: m.get_date() is not None, msgs))
```

Comment:
I think because they will be sorted in the next line and that only works with lists? But then you can also use list comprehension instead of filter().